recipe/meta.yaml

```diff
@@ -9,12 +9,13 @@ source:
   sha256: 7f690b18d35048c15438d6d0571f9045cffbec5907e0b1ccf006f889e3a38c0b
 
 build:
-  number: 3
+  number: 4
   script: "{{ PYTHON }} -m pip install . --no-deps -vv"
 
 requirements:
   host:
     - python
```

Comment:
As conda-forge has dropped Python 2 you could make this `noarch: python`, drop the enum requirement, and require `python >=3.6` (or whatever is appropriate).

Reply:
Thank you, that worked and saved energy!